src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -3724,3 +3724,49 @@ def set_disk_access(cmd, client, parameters, resource_group_name, disk_access_na
                        resource_group_name, disk_access_name, disk_access)
 
 # endregion
+
+def ios8601Compaliant(duration):
+    import re
+    return re.match('^P(\d+Y)?(\d+M)?(\d+D)?(T(\d+H)?(\d+M)?(\d+S)?)?$', duration)
+
+# region install patches
+def install_vm_patches(cmd, resource_group_name, vm_name, maximum_duration, reboot_setting, classifications_to_include=None, kb_numbers_to_include=None, kb_numbers_to_exclude=None,
+                       exclude_kbs_requiring_reboot=None, package_name_masks_to_include=None, package_name_masks_to_exclude=None):
+    from azure.mgmt.compute.models import VirtualMachineInstallPatchesParameters, WindowsParameters, LinuxParameters, VMGuestPatchRebootSetting, VMGuestPatchClassificationWindows, VMGuestPatchClassificationLinux
```

Comment:
This usage is forbidden in Compute. Compute is a multi-API module. Please use a dynamic import method. For example,
```
    DiskEncryptionSet, EncryptionSetIdentity, KeyVaultAndKeyReference, SourceVault = cmd.get_models(
        'DiskEncryptionSet', 'EncryptionSetIdentity', 'KeyVaultAndKeyReference', 'SourceVault')
```